### PR TITLE
Speed up decision stage

### PR DESCRIPTION
**Pull request by carlosmondra**
Uses concurrent requests to create decision invitations. Locally, processing time is reduced by half

### CODE REVIEW

File: openreview/conference/invitation.py [hunk 1]
```diff
@@ -1545,13 +1545,13 @@ def set_meta_review_invitation(self, conference, notes):
...
-        invitations = []
         self.client.post_invitation(DecisionInvitation(conference))
-        for note in tqdm(notes, total=len(notes), desc='set_decision_invitation'):
+        def post_invitation(note):
             invitation = self.client.post_invitation(PaperDecisionInvitation(conference, note))
             self.__update_readers(note, invitation)
-            invitations.append(invitation)
+            return invitation
+
+        invitations = tools.concurrent_requests(post_invitation, notes)
```
Comment (melisabok) [inline openreview/conference/invitation.py:1554]: Thanks for doing this! Is there a way to show a progress bar when doing concurrent requests?

Comment (carlosmondra) [reply openreview/conference/invitation.py:1554]: A progress bar is already implemented within concurrent_requests ([here](https://github.com/openreview/openreview-py/blob/9c775a0e6978aff1966f68f32601c3be3952a232/openreview/tools.py#L34)). So there is no need to add it here.

Comment (melisabok) [reply openreview/conference/invitation.py:1554]: Awesome!

**Review by melisabok — APPROVED** [review]

File: openreview/conference/invitation.py (hunk 1 above)
Comment (melisabok) [reply openreview/conference/invitation.py:1554]: You are invited to do the same with the other stages 🙈

**Pull request merged**